code/API_definitions/network_access_management.yaml

```diff
@@ -50,19 +50,13 @@ info:
     # API Functionality
     This API allows API clients to manage isolated networks and reboot network access devices.
 
-    ## Authorization and authentication
+    ### Authorization and authentication
 
-    [Camara Security and Interoperability Profile](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-Security-Interoperability.md)
-    provides details on how a client requests an access token.
+    The "Camara Security and Interoperability Profile" provides details on how a client requests an access token. Please refer to Identify and Consent Management (https://github.com/camaraproject/IdentityAndConsentManagement/) for the released version of the Profile.
 
-    Which specific authorization flows are to be used will be determined during onboarding process, happening between
-    the API Client and the Telco Operator exposing the API, taking into account the declared purpose for accessing the
-    API, while also being subject to the prevailing legal framework dictated by local legislation.
+    Which specific authorization flows are to be used will be determined during onboarding process, happening between the API Client and the Telco Operator exposing the API, taking into account the declared purpose for accessing the API, while also being subject to the prevailing legal framework dictated by local legislation.

```

Comment:
Should we call it a Network Operator instead of a Telco Operator?

Reply:
Ugh...this is mandatory text...I'll open an issue with ICM.